### DISCUSSION TITLE
[Feedback] Should have a option, Teleport player to Radar won't drop specified items

### DISCUSSION THREAD

**Discussion by CoolLKKPS** [MrHydralisk/Lethal-Company-Enhanced-Radar-Booster#13] [Ideas]
this is for LC_CustomDifficulty, that mod will be really hard if don't have this option

**Comment by MrHydralisk** [comment]
Some mods like Better Teleporter already have this feature and, since ERB mod just changing the end point of inverse teleportation, so that feature from it works for teleporting to Radar Booster too.

**Comment by CoolLKKPS** [comment]
> Some mods like Better Teleporter already have this feature and, since ERB mod just changing the end point of inverse teleportation, so that feature from it works for teleporting to Radar Booster too.

u don't know what i mean, that's for compatibility like backroom mod: inverse teleporter should not keep the items when player enter "level !", and sure backroom did drop all the items, but after leave "level !" u can't grab the items again since the game think u keep those items

**Comment by MrHydralisk** [comment]
But aren't it then suppose to be asked from backroom mod dev then? Cause I don't change or add logic to teleportation itself. Or do I miss smf? Does it needed only for backrooms or other mods?

**Comment by CoolLKKPS** [comment]
yes but i mean, if u can make inverse teleporter can able to teleport items to radar without player to grab it, or make inverse teleporter can only keep items that player have when the target is radar, that will be good bcz u also solve the compatibility issues and i don't need to worry about teleporter mods always keep items without condition, or backroom devs are still not updating to fix the mod

**Comment by MrHydralisk** [comment]
I get what you mean now. Not like I have clue how it decide to drop items during teleportation at the moment. Then different question, why exactly it shouldn't keep items only when TP to Radar Booster, but keep it with random position teleportation? I just have no idea about how or what backrooms mod do.

**Comment by CoolLKKPS** [comment]
i think if u want to teleport player to specify location, that only can done by inverse teleporter mechanism, i don't want to think so much bc im tired, 4 am now and im still working on modpack..

**Comment by CoolLKKPS** [comment]
so in my modpack, u can't exit though the front door, and u can't enter though the fire exit, and more landmine are always actived, and not only that u will get full darkness if u don't have any light source, that mean if u didn't have flashlight, u are done, so must have a way to keep some items into the dungeons after teleportation
BUT the backroom have a bug, if enter "level !", the mod will force u drop every items, and after player back they can't pick up any items that previously in player pocket, so i pretty sus that's teleport mod issues, but after today i tested, no matter install any teleport mod or not, this issue still can happen, so might be just a gameplay feedback here, u don't have to take this seriously
but if u can, i would appreciate that really

**Comment by MrHydralisk** [comment]
1. Correct me if I wrong, when you teleport to Radar Booster (RB) that inside Backrooms, then player keep his items, so when he trying to leave backrooms all this items getting lost inside the Backrooms. 
2. How RB end up in Backrooms in the first place, if you said that you drop all items.
3. Also explain what "level !" mean, cause it mean nothing to me as I never used Backrooms.
4. Did you already reported it to Backrooms dev, cause from what I saw on mod page it seems to be in early development and this bug on their side might be fixed all together with future updates.

**Reply by CoolLKKPS** [reply]
so the first thing i have to say more clearly, bc i also make some mistake:
1. "Level !" is from Diversity mod even similar to backroom, if u killed by bracken, will trigger this mechanism, only 1 player can enter "Level !" in the same round, and only can trigger once
2. u can never able to teleport to "Level !", bc player have to run for own life to paying the price
3. u will drop all the items to enter "Level !"
4. after player are escaped from "Level !", player cannot able to pickup the items that previous dropped
5. for my modpack design, players need to have a light source to explore the Dungeons, otherwise endless darkness will reign, but i don't want players to be able to teleport to secure the loot or just find the way back to the front gate, flashlight only have 5 mins battery, and i set RB can only use inverse tp, i want to force player, if u want to keep some items, u have to use RB, RB will be like a checkpoint
6. BUT actually something happen, RB won't destroy after the ship departed
7. will report "Level !" issue to Diversity mod dev

and bc u don't know backroom, the Backrooms are real-world glitch, and Level 0 is the starting point of the Backrooms, which is a maze, u can break into something like paint or ship to access another level or try to escape, in the Backroom mod, u need to find a room that is decorated much like the interior of the ship to escape; but "level !" is not, u have to run away from monster and break the exit door, it's definitely not a short journey

**Reply by CoolLKKPS** [reply]
DO NOT tp player if they inside the "level !" btw, that will kill ur teammate and the dead body won't return
